addons/payment_authorize/models/payment.py

```diff
@@ -121,7 +121,10 @@ def authorize_form_generate_values(self, values):
 
     def authorize_get_form_action_url(self):
         self.ensure_one()
-        return self._get_authorize_urls(self.environment)['authorize_form_url']
+        if self.state == 'disabled':
+            raise ValidationError(_("The acquirer isn't enabled"))
```

Comment:
again, pbly overkill - and if it had to be done, it should be done in the `render` method on `payment.acquirer` at the global level and not for each implem
remove this error in all implems plz